frontend/packages/console-app/src/components/network-policies/_create-network-policy.scss

```diff
@@ -45,3 +45,7 @@
 .co-create-networkpolicy__rule-header .pf-m-secondary .pf-c-dropdown__toggle-icon {
   margin-right: 0;
 }
+
+.pf-c-tree-view__list-item {
+  list-style: none;
+}
```

Comment:
I did some tests to fix the pointer issue (so that the cursor doesn't suggest clicking on a pod trigger any effect), this CSS would do the trick:

```
.co-create-networkpolicy__selector-preview button.pf-c-tree-view__node[role="treeitem"],
.co-create-networkpolicy__selector-preview button.pf-c-tree-view__node[role="treeitem"] > .pf-c-tree-view__node-container {
  cursor: default;
}
```

Alternatively, patternfly provides a variant named "compactNoBackground" in `TreeView` that also removes the pointer cursor, but weirdly this variant is less compact than the normal one and IMO doesn't look good in our use case, so I'd stick with a CSS override.